styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -91,6 +91,7 @@
   public static final String PROPERTY_CONFIG_DOCKER_RUNNER_ID = "dockerRunnerId";
   public static final String PROPERTY_CONFIG_CONCURRENCY = "concurrency";
   public static final String PROPERTY_CONFIG_CLIENT_BLACKLIST = "clientBlacklist";
+  public static final String PROPERTY_CONFIG_EXECUTION_GATING = "executionGating";
```

Comment:
`PROPERTY_CONFIG_EXECUTION_GATING` naming is not very aligned with other boolean properties, such as `PROPERTY_CONFIG_DEBUG_ENABLED`.